plugins/importexport/onix30/filter/MonographONIX30XmlFilter.php

```diff
@@ -555,8 +547,13 @@ public function createProductNode($doc, $submission, $publicationFormat)
                     $territoryNode->appendChild($this->_buildTextNode($doc, 'RegionsExcluded', trim(join(' ', $salesRights->getRegionsExcluded()))));
                 }
 
-                unset($territoryNode);
-                unset($salesRightsNode);
+                // Include territory and sales rights if the territory isn't empty
+                if ($territoryNode->firstElementChild) {
```

Comment:
At the `DTD` the `<Territory>` node requires a region/country, and the `SalesRights` requires a `<Territory>`.
As the form doesn't ensure the requirements are fulfilled, I decided to implement it this way.